composer.json

```diff
@@ -22,7 +22,7 @@
         "doctrine/inflector": "^1.4|^2.0",
         "dragonmantank/cron-expression": "^3.0.2",
         "egulias/email-validator": "^2.1.10",
-        "league/commonmark": "^1.3",
+        "league/commonmark": "^1.3|^2.0",
```

Comment:
Technically you'll want `^2.0beta3`, as the type hints in the first two betas do not support `$converter->getEnvironment()->addExtension()` as used below.